### PR TITLE
EmptyTemplateHandler fix arguments

### PR DESCRIPTION
**Pull request by ahorek**
fixes https://github.com/rspec/rspec-rails/issues/2086

### CODE REVIEW

File: lib/rspec/rails/view_rendering.rb [hunk 1]
```diff
@@ -102,7 +102,7 @@ def find_templates(*args)
 
       # @private
       class EmptyTemplateHandler
-        def self.call(_template)
+        def self.call(*)
```
Comment (JonRowe) [inline lib/rspec/rails/view_rendering.rb]: It's preferable to have 1 required, 1 optional, just because it matches their API

**Comment by JonRowe** [comment]
If possible can we get a test for this?

**Comment by ahorek** [comment]
there're already about 15 tests failing on rails master, so I don't think it's necessary

https://travis-ci.org/rspec/rspec-rails/jobs/499366086
```
1) GadgetsController GET #edit returns a success response
     Failure/Error: get :edit, params: {id: gadget.to_param}, session: valid_session
     
     ActionView::Template::Error:
       wrong number of arguments (given 2, expected 1)
     # /home/travis/build/rspec/bundle/ruby/2.6.0/bundler/gems/rails-controller-testing-21014e48be12/lib/rails/controller/testing/template_assertions.rb:61:in `process'
     # /home/travis/build/rspec/bundle/ruby/2.6.0/bundler/gems/rails-controller-testing-21014e48be12/lib/rails/controller/testing/integration.rb:13:in `block (2 levels) in <module:Integration>'
     # ./spec/controllers/gadgets_controller_spec.rb:70:in `block (3 levels) in <main>'
     # ------------------
     # --- Caused by: ---
     # ArgumentError:
     #   wrong number of arguments (given 2, expected 1)
     #   /home/travis/build/rspec/bundle/ruby/2.6.0/bundler/gems/rails-controller-testing-21014e48be12/lib/rails/controller/testing/template_assertions.rb:61:in `process'
```

**Comment by JonRowe** [comment]
Fair enough

**Comment by benoittgt** [comment]
Didn't see it. Thanks!

**Comment by nijikon** [comment]
Can we backport this to 3.x and release a new version? /cc @JonRowe

**Comment by JonRowe** [comment]
Sorry no, work to release 4.0.0 is under way and it will be released soon, but its too much maintenance work to support Rails 6 in 3.x.

**Pull request merged**